rundeckapp/grails-spa/packages/ui-trellis/src/library/components/execution-log/logViewer.vue

```diff
@@ -49,9 +49,11 @@
           <input type="checkbox" v-model="settings.stats" id="logview_stats">
           <label for="logview_stats">Display Stats</label>
         </div>
-        <ui-socket section="execution-log-viewer" location="settings"
-                   :event-bus="eventBus"
-        />
+        <div class="vue-ui-socket">
+          <ui-socket section="execution-log-viewer" location="settings"
+                     :event-bus="eventBus"
+          />
+        </div>
       </form>
```

Comment:
why this change, did it affect things? i don't think we should wrap ui-socket like that when it is already inside a Vue component

Reply:
Just updated it to match the format recommended for the vue 3 migration (ui-socket inside of another element with vue-ui-socket class) but it doesn't make any difference how things are rendered in practice

Reply:
I can undo it if you'd like

Reply:
yes, that format is only needed when embedding ui-socket into the page outside of a Vue component